core/core.js

```diff
@@ -208,14 +208,14 @@ SharedQueue Binary Layout
   }
 
   function decodeJson(ui8) {
-    const s = Deno.core.decode(ui8);
```

Comment:
This binding made json dispatch defined in this file work only when `Deno` global was available - so not working in workers by default